keyboard_teleop:
  speed: 5.0
  max_steering_angle: 0.34
